src/Codeception/Command/Console.php

```diff
@@ -81,6 +81,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $this->suite    = $suiteManager->getSuite();
 
         $scenario = new Scenario($this->test);
+        if (isset($config["namespace"])) $settings['class_name'] = $config["namespace"].'\\'.$settings['class_name'];
         $actor      = $settings['class_name'];
```

Comment:
Don't think so, class name includes a namespace. Property is unusable without the namespace.